index.html

```diff
@@ -7,14 +7,202 @@
+                     </li >
+                     <li class = "nav-item">
+                        <a class = "nav-link" href = "#">Blog</a>
+                     </li >
+                     <li class = "nav-item">
+                        <a class = "nav-link" href = "#">Contact</a>
+                     </li >
+                </ul>
+            </div>
+        </nav>
+    </div>
+    <div class="container text-center g-0">
+        <div class="row">
+            <div class="col-md-6 order-md-5 g-0">
+                <img class = "img-fluid" id="flower" src = "./img/green-flower.png" alt = "Green-Blue Flower">
+            </div>
+            <div class="col-md-6 order-md-1 g-0">
+                <div class = introduction>
+                    <h1 class = "page-title">
+                        Online Therapy<br>for Emotional<br>Health
+                    </h1>
+                    <!--Intro-->
+                    <p class = "description">
+                        I am a description. Click here to edit.
+                    </p>
+                    <!--Scroll down arrow-->
+                    <a href = "#">&darr;</a>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class = "container text-center g-0">
+        <div class = "row">
+            <div class = "col g-0">
+                <!--About Section-->
+                <section class = "about-maria">
+                    <h3 class = "About">About</h3>
+                    <h2 class = "Psych-name">Maria Hope</h2>
+                    <p>
+                        I'm a paragraph. Click here to add your own text and edit me. It's easy. Just click "Edit Text" or double click me to add your own content and make changes to the font.
+                        Feel free to drag and drop me anywhere you like on your page. I'm a great place for you to tell a story and let your users know a little more about you.
+                    </p>
+                    <a href = "#">&rarr;</a>
+                </section>
+            </div>
+        </div>
+        <div class = "row">
+            <div class = "col g-0">
+                <img class = "maria-photo img-fluid" src = "./img/maria-hope.png" alt = "Maria Hope">
+            </div>
+        </div>
+    </div>
+    <!--Therapies Section-->
+    <div class = "container text-center g-0">
+        <section class = "therapies">
+        <div class = "row">
+            <div class = "col-md-4 g-0">
+                <section class = "individual">
+                    <h3>01.</h3>
+                    <h2>Individual Therapy</h2>
+                    <p>
+                        I'm a paragraph. Click here to add your own text and edit me. It's easy. Just click "Edit Text" or double click me to add your own content and make changes to the font.
+                    </p>
+                </section>
+            </div>
+            <div class = "col-md-4 g-0">
```

Comment:
This seems to work fine here but FYI I believe you can apply the `g-0` class directly on the `row` so you don't have to repeat it on each `col-...` element